app/res/xml/commcare_preferences.xml

```diff
@@ -32,7 +32,7 @@
         android:defaultValue="no"
         android:entries="@array/pref_enabled_labels"
         android:entryValues="@array/pref_enabled_vals"
-        android:key="cc-newest-version-from-hq"
+        android:key="cc-update-to-unstarred-builds"
```

Comment:
I'm wondering if there is any way that this preference key change could cause issues for people who have already enabled this feature. Did you check this?